Endpoints.swift

```diff
@@ -12,32 +12,39 @@ import SMART
 
 func configuredEndpoints() -> [Endpoint] {
 	var endpoints = [Endpoint]()
-	
-	// Grahame's test server
-	let grahame = Client(
-		baseURL: URL(string: "http://fhir3.healthintersections.com.au/open")!,
-		settings: [
-			"client_name": "SMART on FHIR iOS Sample App",
-			"redirect": "smartapp://callback",
-			"logo_uri": "https://avatars1.githubusercontent.com/u/7401080",
-			])
-	grahame.authProperties.granularity = .patientSelectNative
-	grahame.authProperties.embedded = true
-	endpoints.append(Endpoint(client: grahame, name: "Health Intersections"))
-	
-	// SMART DSTU-2!! sandbox
-	// Credentials obtained by registering on the SMART website
-	let smart = Client(
```

Comment:
Also keep SMART – we'll soon have an STU-3 version and will only need to update the URL. It's fine if not all servers work out of the box, and it's good to have the stubs in code already.